tools/path/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import path from 'path'
+
+export function getToolDirPath (
+  opts: {
+    pnpmHomeDir: string
+    tool: {
+      name: string
+      version: string
+    }
+  }
+): string {
+  return path.join(opts.pnpmHomeDir, '.tools', opts.tool.name.replaceAll('/', '+'), opts.tool.version)
```

Comment:
Curious if `.tools` needs the leading dot since it's nested under the pnpm home directory?

Reply:
Yes, because we use the pnpm home directory for the global binaries too, so to avoid any collisions, we use the dot